### PR TITLE
Ft: Azure Obj Copy

### PR DESCRIPTION
**Pull request by dora-korpar**

### CODE REVIEW

File: tests/functional/aws-node-sdk/test/multipleBackend/objectCopy/azureObjectCopy.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    require('../../../lib/utility/createEncryptedBucket');
+
+const azureLocation = 'azuretest';
+const azureClient = getAzureClient();
+const azureContainerName = getAzureContainerName();
+
+const bucket = 'buckettestmultiplebackendobjectcopy';
+const key = `azureputkey-${Date.now()}`;
+const copyKey = `azurecopyKey-${Date.now()}`;
+const body = Buffer.from('I am a body', 'utf8');
```
Comment (LaurenSpiegel) [inline tests/functional/aws-node-sdk/test/multipleBackend/objectCopy/azureObjectCopy.js]: let's also test with a bigger body so we can make sure the streaming functionality is working correctly.  maybe 5 mb?

File: tests/functional/aws-node-sdk/test/multipleBackend/objectCopy/azureObjectCopy.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+                    assert.equal(err, null, 'Expected success but got ' +
+                    `error: ${err}`);
+                    assert.strictEqual(result.CopyObjectResult.ETag,
+                        `"${emptyMD5}"`);
+                    assertGetObjects(key, bucket, azureLocation, copyKey,
+                        bucket, azureLocation, copyKey, 'COPY', true, done);
+                });
+            });
+        });
+
```
Comment (LaurenSpiegel) [inline tests/functional/aws-node-sdk/test/multipleBackend/objectCopy/azureObjectCopy.js]: should we get fancy and test aws to azure and azure to aws?

Comment (dora-korpar) [inline tests/functional/aws-node-sdk/test/multipleBackend/objectCopy/azureObjectCopy.js]: oh I didn't even think of that. can do

**Comment by LaurenSpiegel** [comment]
@dora-korpar, encryption pr was merged.  please rebase this and see if the problem you were having still exists.

**Comment by dora-korpar** [comment]
@ironman-machine try

**Pull request closed without merging**